packages/contracts/contracts/TwoPartyVirtualEthAsLump.sol

```diff
@@ -4,16 +4,14 @@ pragma experimental "ABIEncoderV2";
 import "./AppRegistry.sol";
 import "./NonceRegistry.sol";
 
-/// todo(xuanji): rename to TwoPartyVirtualEthAsLump
 
-
-/// @title ETHVirtualAppAgreement
+/// @title TwoPartyVirtualEthAsLump
 /// @notice Commitment target to support "virtual apps", i.e., apps which have
 /// ETH committed to them via intermediary lock-up instead of having ETH directly
 /// committed in a ledger channel.
 /// The `target` contract must return via resolution the outcome of the app,
 /// and the resolve function must be TwoPartyOutcome type.
-contract ETHVirtualAppAgreement {
+contract TwoPartyVirtualEthAsLump {
```

Comment:
Can you write briefly, perhaps as a comment on the pull-request, _why_ we need this file versus re-using the `StateChannelTransaction.sol` file? Of course I know, but it is an interesting point worth explaining